src/sb_utils/include/RvizUtils.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: November 10th, 2017
+ * Description: Utilities to create Rviz visualisation messages
+ *
+ */
+#ifndef SB_UTILS_RVIZUTILS_H
+#define SB_UTILS_RVIZUTILS_H
+
+// Messages
+#include <geometry_msgs/Point.h>
+#include <visualization_msgs/Marker.h>
+
+using namespace std;
+using namespace visualization_msgs;
+
+class RvizUtils {
+public:
+    /**
+     * Turn the points into a marker for rviz
+     *
+     * @param points the points to be converted
+     * @param color the color of the points
+     */
+    static Marker displayPoints(vector<geometry_msgs::Point> points, Marker::_color_type color,
+                                Marker::_scale_type scale, string frame_id, string ns);
+
+    /**
+     * Turn the point into a marker for rviz
```

Comment:
nit-pick, but this should say `a point`, rather then `the point`

Reply:
Fixed